avm/res/network/connection/tests/e2e/vnet2vnet/main.test.bicep

```diff
@@ -0,0 +1,74 @@
+targetScope = 'subscription'
+
+metadata name = 'Using VNet-to-VNet parameter set'
+metadata description = 'This instance deploys the module with the VNet-to-VNet set of required parameters.'
```

Comment:
If you say 'required' here, I have to ask if the bgb parameter must be specified with false for VNET to VNET

Reply:
test folder removed